### DISCUSSION TITLE
Show the waster entities to using the Tile Card from home assistant

### DISCUSSION THREAD

**Discussion by sim186** [mampfes/hacs_waste_collection_schedule#910] [Q&A]
Hi,

currenlty I'm using the custom card to show this:
<img width="519" alt="image" src="https://user-images.githubusercontent.com/18590835/222544975-8e769f07-fa8d-44c0-853a-e4c5a65174f9.png">

I was wondering if there is a way to use the new tile cards from home assistant.
https://www.home-assistant.io/dashboards/tile/

Thanks.

**Comment by dt215git** [comment]
The Tile Card looks easy enough to set up
![image](https://user-images.githubusercontent.com/86194065/225111155-b0855956-6bc1-4c74-bc85-b66a7d44ffca.png)
But you'll probably need set up the background actions you want executed when you click on it. Did you have any idea what you were wanting to do - change the icon color, use a different image, make it hidden until the daysTo is less that 3 days, etc?

**Comment by sim186** [comment]
Exactly. You are right, what I meant is not only the icon and the text but also the color change based on the left days. I think without custom card is not possible right?

**Reply by Rattleheadie** [reply]
Hey, I hope you dont mind me digging up an old post but for the love of all that is good, how did you get the above working or did you ever get the "color depending on what garbage is next" to work. I cant even get colors on my icons, its really frustrating me, thanks

**Comment by dt215git** [comment]
Doesn't look like the Tile Card supports templating at the moment. If it did, something like this might have worked.

```yaml
type: tile
entity: sensor.<<NameGoesHere>>
name: Next Collection
vertical: true
icon: mdi:trash-can
color: >
    {% if state_attr("sensor.<<NameGoesHere>>", "daysTo") | int == 6 %}
        blue
    {% elif state_attr("sensor.<<NameGoesHere>>", "daysTo") | int == 5 %}
        pink
    {% elif state_attr("sensor.<<NameGoesHere>>", "daysTo") | int == 4 %}
        yellow
    {% elif state_attr("sensor.<<NameGoesHere>>", "daysTo") | int == 3 %}
        red
    {% elif state_attr("sensor.<<NameGoesHere>>", "daysTo") | int == 2 %}
        orange
    {% elif state_attr("sensor.<<NameGoesHere>>", "daysTo") | int == 1 %}
        green
    {% else %}
        black
    {% endif %}
```

Maybe that gives you some idea of what you could try with the Custom Card?
